[DOCS] Trusted Apps by Policy (#1240)
elastic/security-docs: docs/management/admin/trusted-apps.asciidoc

* `Name your trusted application`: Enter a name for the trusted application.

* `Description`(Optional): Enter a description for the trusted application.

* `Select operating system`: Select the appropriate operating system from the drop-down.

* `Field`: Select the appropriate field you want to use -- `Hash`, `Path`, or (if you're adding a trusted application on Windows) `Signature`.
+

> NOTE: You can only add a single field type value per trusted application. For example, if you try to add two `Path` values, you'll get an error message. Hash values must also be valid to add the trusted application. In addition, to minimize the gaps in visibility to the Security solution, be as specific as possible in your entries. For example, combine `Signature` information with known `Paths`.

Reviewer: ```suggestion
NOTE: You can only add a single field type value per trusted application. For example, if you try to add two `Path` values, you'll get an error message. Hash values must also be valid to add the trusted application. In addition, to minimize visibility gaps in the {security-app}, be as specific as possible in your entries. For example, combine `Signature` information with known `Paths`.
```

+

* `Operator`: Select an operator to define the condition:
   - `is`: Must be exactly equal to `Value`. This operation is required for the `Hash` and `Signature` field types.   
   - `matches`: Can include wildcards in `Value`, such as `C:\path\*\app.exe`. This option is only available for the `Path` field type.

* `Value`: Enter the hash value or file path. To add an additional value, click *AND*.

. Select an option in the *Assignment* section to assign the trusted application to a specific integration policy:
+
* `Global`: Assign the trusted application to all integration policies for {endpoint-sec}.
* `Per Policy` (Platinum+ subscription only): Assign the trusted application to one or more specific {endpoint-sec} integration policies. Select each policy in which you want the application to be trusted.
+

> NOTE: You can also select the `Per Policy` option without assigning a policy to the trusted application at this time. For example, you might want to create and review your trusted application configurations before putting them into action with a policy.

Reviewer 2: ```suggestion
NOTE: You can also select the `Per Policy` option without assigning a policy to the trusted application at this time. For example, you could select the `Per Policy` option if you wanted to create and review your trusted application configurations before putting them into action with a policy.
```

Author: How about this small modification -- it's not just selecting `Per Policy`, but it's doing that _and_ not selecting any policies. The first sentence already says those two things, so we could just reference that.
```suggestion
NOTE: You can also select the `Per Policy` option without assigning a policy to the trusted application at this time. For example, you could do this if you wanted to create and review your trusted application configurations before putting them into action with a policy.
```

Reviewer 2: Works for me!

> . Click *Add trusted application*. The application is added to the Trusted applications list.

Reviewer 2: ```suggestion
. Click *Add trusted application*. The application is added to the *Trusted applications* list.
```

[discrete]
[[trusted-apps-list]]
== View and manage trusted applications

> The *Trusted applications* list displays all the trusted applications that have been added to the {security-app}. To refine the Trusted applications list, enter a query in the search bar. You can search by name, description, or the value of a field.

Reviewer 2: ```suggestion
The *Trusted applications* list displays all the trusted applications that have been added to the {security-app}. To refine the *Trusted applications* list, enter a query in the search bar. You can search by name, description, or the value of a field.
```

> The *Trusted applications* list displays all the trusted applications that have been added to the {security-app}. To refine the *Trusted applications* list, enter a query in the search bar. You can search by name, description, or the value of a field.

Reviewer: ```suggestion
The *Trusted applications* list displays all the trusted applications that have been added to the {security-app}. To refine the *Trusted applications* list, enter a query in the search bar. You can search by name, description, or a field value.
```
```suggestion
The *Trusted applications* list displays all the trusted applications that have been added to the {security-app}. To refine the *Trusted applications* list, enter a query in the search bar. You can search by name, description, or the value of a field.
```

[role="screenshot"]
image::images/trusted-apps-list.png[]

[discrete]
[[edit-trusted-app]]
=== Edit a trusted application

> You can change the configuration of each trusted application. If you have a Platinum+ subscription, you can also change the policy assigned to a trusted application.

Reviewer 2: ```suggestion
You can change the configuration of each trusted application. If you have a Platinum subscription, you can also change the policy assigned to a trusted application.
```

* `Name your trusted application`: Enter a name for the trusted application.

* `Description`(Optional): Enter a description for the trusted application.

* `Select operating system`: Select the appropriate operating system from the drop-down.

* `Field`: Select the appropriate field you want to use -- `Hash`, `Path`, or (if you are adding a trusted application on Windows) `Signature`.
+
NOTE: You can only add a single field type value per trusted application. For example, if you try to add two `Path` values, you'll get an error message. Hash values must also be valid to add the trusted application. In addition, to minimize the gaps in visibility to the Security solution, be as specific as possible in your entries. For example, combine `Signature` information with known `Paths`.

> +

Reviewer 3: Note: the Operator can be `is` or `matches` if field is `path`

Reviewer 4: good catch @dasansol92 

@joepeeples can we add a note here for this distinction?

* For `path` only, users can choose `is` or `matches`.  If users pick `is` then the full path must match exactly.  If users pick `matches` they can include wildcards such as `C:\some_path\*\program.exe`

Author: @dasansol92 @kevinlog Thanks for catching this! I noticed `matches` per https://github.com/elastic/security-docs/issues/1243, but I thought that wasn't released until 8.0. But if it's in 7.16, we'll definitely want to add to docs.

Reviewer 4: Thanks @joepeeples - I can see the change below.  I updated the ticket to be marked `7.16` and we can close it with this PR.

https://github.com/elastic/security-docs/issues/1243

* `Value`: Enter the hash value or file path. To add an additional value, click *AND*.

. Select an option in the *Assignment* section to assign the trusted application to a specific integration policy:
+
* `Global`: Assign the trusted application to all integration policies for {endpoint-sec}.

> * `Per Policy`: Assign the trusted application to one or more specific {endpoint-sec} integration policies. Select each policy in which you want the application to be trusted.

Reviewer 4: we should add another `NOTE:` here that users are able to assign a Trusted App to 0 policies in the per policy section.  This could be useful if users want to create and review their Trusted Apps first and slowly roll them out to policies.
